cfgov/jinja2/v1/ask-cfpb/landing-page-spanish.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'spanish-ask-base.html' %}
+
+{% block babel_content %}
+<section class="content full-bleed">
+    <div class="primary">                
+        <div class="ac-catnav-row">              
+        {% for cat in categories %}
+            <article class="ac-catnav ac-cat-{{ cat.slug_es }}">
+                <h2 class="ac-catnav-heading">
+                    <a href="">
```

Comment:
@virginiacc Make sure to keep track of these empty `href`s so they get fixed before launch.